sig-etcd/README.md

```diff
@@ -114,5 +114,6 @@ Serve the Apache Zookeeper API but back it with an etcd cluster
 [subproject-definition]: https://github.com/kubernetes/community/blob/master/governance.md#subprojects
 [working-group-definition]: https://github.com/kubernetes/community/blob/master/governance.md#working-groups
 <!-- BEGIN CUSTOM CONTENT -->
-
+## Meetings
```

Comment:
Shouldn't this go under the Meetings section that already exists?

Reply:
the checks are failing if i do it.

Reply:
Yeah, because you're trying to do this through Custom Content instead of adding it to sigs.yaml, where it belongs.

It is not acceptable to make this modification this way.